Program.cs

```diff
@@ -13,23 +13,16 @@ static void Main()
 		{
 			CultureInfo.DefaultThreadCurrentCulture = CultureInfo.InvariantCulture;
 
-			try
-			{
-				var settings = Settings.Load(SettingsFile);
-
-				using (var nativeHelper = new NativeHelper())
-				{
-					Application.EnableVisualStyles();
-					Application.SetCompatibleTextRenderingDefault(false);
-					Application.Run(new MainForm(nativeHelper, settings));
-				}
+			var settings = Settings.Load(SettingsFile);
 
-				Settings.Save(settings, SettingsFile);
-			}
-			catch (Exception ex)
+			using (var nativeHelper = new NativeHelper())
 			{
-
+				Application.EnableVisualStyles();
+				Application.SetCompatibleTextRenderingDefault(false);
+				Application.Run(new MainForm(nativeHelper, settings));
```

Comment:
I have tested it again and this time the exception was shown at the source. Don't know what was the problem before. Anyway I have added the global try catch only in release mode now.